media-plugins/vdr-osdteletext/vdr-osdteletext-0.9.7.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit user vdr-plugin-2
+
+VERSION="2161" # every bump, new version
+
+DESCRIPTION="VDR Plugin: Osd-Teletext displays the teletext on the OSD"
+HOMEPAGE="https://projects.vdr-developer.org/projects/plg-osdteletext"
+SRC_URI="https://projects.vdr-developer.org/attachments/download/${VERSION}/${P}.tgz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="media-video/vdr"
+RDEPEND="${DEPEND}"
+
+VDR_RCADDON_FILE="${FILESDIR}/rc-addon-v3.sh"
+VDR_CONFD_FILE="${FILESDIR}/confd-v2"
+
+QA_FLAGS_IGNORED="
+	usr/lib/vdr/plugins/libvdr-.*
+	usr/lib64/vdr/plugins/libvdr-.*"
+
+src_install() {
+	vdr-plugin-2_src_install
+
+	insinto /etc/sudoers.d
+	insopts -m440
+	newins "${FILESDIR}/vdr-osdteletext.sudo" vdr-osdteletext
+
+	local vdr_user_home=$(egethome vdr)
```

Comment:
Thinking about it, just hardcode the correct path. It's hardcoded in gentoo-vdr-scripts anyway, and when it's migrated to acct-* it will be unchangeable.

Reply:
Sure? I hoped to change vdr's $HOME one day to a FHS conformant `/var/lib/vdr` and therefore planned to remove the hardcoded `/var/vdr` from gentoo-vdr-scripts. 
I did some investigation with a private acct-user/vdr and found out it will enforce a change of the home directory - I expected/hoped a behaviour to keep $HOME when the user previously exists, like keeping the existing UID/GID. This will enable the admin to migrate both UID/GID and $HOME if he wants - or not. And new installations would be standardized.

Reply:
But if you think 
> when it's migrated to acct-* it will be unchangeable
then I hardcode it....